src/projection/node/create-projection-node.ts

```diff
@@ -351,12 +353,24 @@ export function createProjectionNode<I>({
                 this.isLayoutDirty = true
             }
 
-            attachResizeListener?.(instance, () => {
-                if (globalProjectionState.hasAnimatedSinceResize) {
-                    globalProjectionState.hasAnimatedSinceResize = false
-                    this.nodes!.forEach(finishAnimation)
-                }
-            })
+            if (attachResizeListener) {
+                let unblockTimeout: number
+
+                const unblockUpdate = () =>
+                    (this.root.updateBlockedByResize = false)
```

Comment:
There can be confusion between this and `this.unblockUpdate` can we rename this to something more specific? `resizeUblockUpdate` perhaps?